manifests/0000_70_dns-operator_03-cluster-operator.yaml

```diff
@@ -5,3 +5,11 @@ apiVersion: config.openshift.io/v1
 kind: ClusterOperator
 metadata:
   name: dns
+status:
+  versions:
+    - name: operator
+      version: "0.0.1-snapshot"
+    - name: coredns
+      version: "0.0.1-snapshot"
+    - name: operator-cli
+      version: "0.0.1-snapshot"
```

Comment:
[The CVO documentation](https://github.com/openshift/cluster-version-operator/blob/8402d219f36fc79e03edf45918785376113f2cc1/docs/dev/clusteroperator.md#what-should-be-the-contents-of-clusteroperator-custom-resource-in-manifests) says only to add an entry for the operator itself.

Reply:
Corroborating the documentation, Derek's original PR only added an entry for the operator: https://github.com/openshift/cluster-dns-operator/pull/78/files#diff-cc9f42cdd0bb164691de71f425fd6f6eR8

Reply:
Fixed